x-pack/plugin/security/qa/saml-rest-tests/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'elasticsearch.internal-java-rest-test'
+apply plugin: 'elasticsearch.internal-test-artifact'
```

Comment:
Some utility code from this test is used in the plugin. An option is to move this to a shared location.